elasticsearch_metrics/field.py

```diff
@@ -0,0 +1,27 @@
+from django.conf import settings
+from elasticsearch_dsl import field as edsl_field
+
+__all__ = ["Date"]
+# Expose all fields from elasticsearch_dsl.field
+# We do this instead of 'from elasticsearch_dsl.field import *' because elasticsearch_metrics
+# has its own subclass of Date
+for each in (
+    field_name
+    for field_name in dir(edsl_field)
+    if field_name != "Date" and not field_name.startswith("_")
+):
+    field = getattr(edsl_field, each)
+    is_field_subclass = isinstance(field, type) and issubclass(field, edsl_field.Field)
+    if field is edsl_field.Field or is_field_subclass:
+        __all__.append(each)
+        globals()[each] = field
```

Comment:
Should we also overwrite` __module__ `here on the field? `field.__module__ = each`.  Just poking around this [old commit ](https://github.com/elastic/elasticsearch-dsl-py/commit/03fc51a954576519fee94f0cc5515c9ed1dd2cdf#diff-38cde0cba5a188e99558e5db2114aec5)

Reply:
Ah, good catch. I'll add `field.__module__ = __name__`.